fluentlenium-core/src/test/java/org/fluentlenium/integration/localtest/LocalFluentCase.java

```diff
@@ -29,7 +29,7 @@ public WebDriver getDefaultDriver() {
         return new HtmlUnitDriver(true);
     }
     public static String getPath() {
-        String currentDir = System.getProperty("user.dir");
+        String currentDir = System.getProperty("user.dir").replace('\\', '/');
```

Comment:
As in previous classes I don't like such replacement. What is a reason to having that in the code?